pkg/apis/contrail/v1alpha1/config_types.go

```diff
@@ -108,6 +107,8 @@ type ConfigStatus struct {
 	Ports         ConfigStatusPorts                 `json:"ports,omitempty"`
 	ConfigChanged *bool                             `json:"configChanged,omitempty"`
 	ServiceStatus map[string]ConfigServiceStatusMap `json:"serviceStatus,omitempty"`
+	ConfigAPIURL  string                            `json:"configapiurl,omitempty"`
```

Comment:
Maybe we could only provide ClusterIP, here. Other controllers can create URLs on their own using "ClusterIP" field and ports from "Ports"

Reply:
Providing only clusterIP now